custom/opm/reports.py

```diff
@@ -434,7 +434,7 @@ def _service_dates(self):
         for (owner_id, date), row in data.iteritems():
             if row['vhnd_available'] > 0:
                 for prop in VHND_PROPERTIES:
-                    if row[prop] == '1' or prop == 'vhnd_available':
+                    if row[prop] == 1 or prop == 'vhnd_available':
```

Comment:
Only reason I can think why it might be like that is if 'vhnd_available' uses a different value to represent "true", or can be other integers for some crazy reason.  If that and all the others just use the integer `1`, I'd be in favor of removing that extra clause.